netci.groovy

```diff
@@ -436,7 +436,7 @@ def static addCopyCoreClrAndRunTestSteps(def job, def coreclrBranch, String os,
             }
 
             // Unpack the build data
-            shell("unpacker ./bin/build.pack ./bin")
+            shell("unpacker ./bin/build.pack .")
             // Export the LTTNG environment variable and then run the tests
```

Comment:
why unpack to workspace root? wouldn't this fail the subsequent run-test.sh with .\bin\tests.. or was the packer modified to drop this to .\bin\---, etc.. Also, we are still using run-test.sh, don't understand why we are passing the xunit args to build.cmd..